starboard/linux/x64x11/BUILD.gn

```diff
@@ -30,6 +30,7 @@ static_library("starboard_platform") {
 
 if (build_with_separate_cobalt_toolchain) {
   static_library("starboard_platform_with_main") {
+    check_includes = false
```

Comment:
do you want to ahve a todo here to remove this ?

curious as to how this fails

Reply:
same as in starboard_platform, I don't think it'll be that easy to solve (or all that necessary)